images/linux/scripts/helpers/Tests.Helpers.psm1

```diff
@@ -0,0 +1,72 @@
+Import-Module "$PSScriptRoot/../helpers/Common.Helpers.psm1" -DisableNameChecking
+
+function Validate-ZeroExitCode($command) {
+    $result = Get-CommandResult $command
+    $result.ExitCode | Should -Be 0 -Because $result.Output
+}
```

Comment:
Looks like this function is some kind of legacy. We should use `Should -ReturnZeroExitCode` instead.
Let's remove it to avoid confusion

Reply:
Yes, I agree, removed it.